tests/pos/i2570.scala

```diff
@@ -0,0 +1,29 @@
+object Test {
+
+  def repeat(s: String, i: Int, j: Int = 22) = s * i
```

Comment:
When discussing this with Seth, he suggested to play around with currying by adding another argument list. The partial application should be fine: `repeat(_, 1)`, but what about `repeat(_, 1)(_)`?  (uncurrying?)